### PR TITLE
CHANGELOG 3.2.1

### PR DESCRIPTION
**Pull request by rachely3n**
Releasing changes as outlined in CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,23 @@
...
+### Fixed
+- Issue #106 Fixed bug where it was assumed default subnetwork under default network was also default.
+- Issue #114 Fix bug in windows config where it was assumed the windows-related instance configuration was first configuration
+under ComputeEngineCloud.
+- Made windows configuration in instance configuration non-transient to make it serializable.
+
+### Changed
+- Issue #65 Replaced public getters and setters with lombok annotations.
+- Issue #104 Moved check Java command after copying agent.jar during instance launch.
+- Issue #108 Removed "Use Internal IP?" from advanced tab of instance configuration.
```
Comment (stephenashank) [inline CHANGELOG.md]: Maybe to make this more clear:

Moved "Use Internal IP?" from *Networking* section under advanced instance configuration to *Launch Configuration* section.

File: pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
   </parent>
 
   <artifactId>google-compute-engine</artifactId>
-  <version>3.2.1-SNAPSHOT</version>
+  <version>3.2.2-SNAPSHOT</version>
```
Comment (stephenashank) [inline pom.xml]: If you're releasing this as 3.2.1 then this should remain as 3.2.1, the maven release plugin will handle the version bump for you.

File: CHANGELOG.md (hunk 1 above)
Comment (rachely3n) [reply CHANGELOG.md]: Done

File: pom.xml (hunk 2 above)
Comment (rachely3n) [reply pom.xml]: Updated to 3.3.0

**Pull request merged**